src/main/java/ch/njol/skript/classes/Comparator.java

```diff
@@ -18,60 +18,52 @@
  */
 package ch.njol.skript.classes;
 
-import ch.njol.skript.classes.data.DefaultComparators;
-import ch.njol.skript.registrations.Comparators;
-
 /**
  * Used to compare two objects of a different or the same type.
- * 
- * @author Peter Güttinger
- * @param <T1> ,
- * @param <T2> the types to compare
- * @see Comparators#registerComparator(Class, Class, Comparator)
- * @see DefaultComparators
+ * @deprecated Use {@link org.skriptlang.skript.lang.comparator.Comparators}
  */
 public interface Comparator<T1, T2> {
```

Comment:
You also need to add the Deprecated annotation (not sure if it's needed explicitly on inner classes)